tests/queries/0_stateless/02864_statistics_estimate_predicate.sql

```diff
@@ -0,0 +1,98 @@
+-- Tags: no-fasttest
```

Comment:
why no fast test?

Reply:
I think it is because `datasketches` is not one of the submodules that are checked out and available in FastTest: https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/fasttest/run.sh#L122

Reply:
well, in this case we can remain this test `tests/queries/0_stateless/02864_statistics_uniq.sql`